src/main/resources/xld.Folder/unpack_create.sh.ftl

```diff
@@ -13,7 +13,7 @@
 #!/bin/bash
 set -e
 
-FILE_TYPE=`file --mime-type -b ${deployed.file.path}`
+FILE_TYPE=`file -i ${deployed.file.path} | sed -n 's/.*: \(.*\);.*/\1/p'`
```

Comment:
Sorry - scratch that. I was on a host that apparently has an excessively old version of the command, that wouldn't even work for the case that I've provided. The -b option is available on file-4.17, so I will update this PR with your suggestion.